lightly/models/modules/masked_vision_transformer.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+
+class MaskedVisionTransformer(ABC):
+    def __init__(self):
+        pass
+
+    @abstractmethod
+    def forward(self):
```

Comment:
The functions in the ABC class should have the same arguments and return types as the functions in `MaskedVisionTransformerTIMM`.